### ISSUE TITLE
Feature Request https basic auth

### ISSUE THREAD

**Issue by chb567** [sawhney17/logseq-calendars-plugin#19]
I wish to add my calendars from my radicale instance that's secured behind basic auth.

Axios allows specifying an auth parameter, so i think this should do the trick:

```
const res = await axios.get('https://httpbin.org/basic-auth/foo/bar', {
  // Axios looks for the `auth` option, and, if it is set, formats a
  // basic auth header for you automatically.
  auth: {
    username: 'foo',
    password: 'bar'
  }
});
res.status; // 200
```

Thanks in advance

**Comment by sawhney17** [comment]
Are you still interested in this feature?

**Comment by chb567** [comment]
Yes, absolutely

**Comment by sawhney17** [comment]
Would it be possible for you to supply a basic calendar ICS link? Just with a few events scheduled for today. It’d really help me make this possible.

**Comment by chb567** [comment]
Certainly. I can give you the link and the credentials, but I'd rather not share it so publicly :P if you could send me an email I can respond with all that's necessary. 
init567@gmx.de

**Comment by aadilayub** [comment]
Highly interested in this feature. Most people with corporate calendar servers have to use auth to access their day's agenda

**Comment by fraschm1998** [comment]
Also very interested!
